errorHandling.js

```diff
@@ -0,0 +1,8 @@
+exports.invalidEndpoint = (request, response) => {
+  response.status(404).send({status: 404, message: "404 - Not Found"})
+}
+
+
+exports.lastErrorHandler = (error, request, response, next) => {
+  response.status(500).send({status: 500, message: "500 - Internal Server Error"})
+}
```

Comment:
I would perhaps change the name of this - it should reflect that it's a 500 error! :)